### PR TITLE
add release log

### PR DESCRIPTION
**Pull request by WintonChan**
Signed-off-by: WintonChan <cw_hao@163.com>

**What type of PR is this?**

/kind documentation

**What this PR does / why we need it**:

add release log

**Which issue(s) this PR fixes**:
Fixes #

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

```release-note

```

### CODE REVIEW

File: CHANGELOG/CHANGELOG-1.7.md [hunk 1]
```diff
@@ -91,7 +91,7 @@ NA
 - Add config of cloudcore token refresh frequence ([#2796](https://github.com/kubeedge/kubeedge/pull/2796), [@leofang94](https://github.com/leofang94))
 - keadm: install CRDs corresponding to specific version ([#2803](https://github.com/kubeedge/kubeedge/pull/2803), [@daixiang0](https://github.com/daixiang0))
 - make customsiz labels available when restart ([#2839](https://github.com/kubeedge/kubeedge/pull/2839), [@ttlv](https://github.com/ttlv))
-
+- add router servicebus ([#2588](https://github.com/kubeedge/kubeedge/pull/2588), [@WintonChan](https://github.com/WintonChan))
```
Comment (fisherxu) [inline CHANGELOG/CHANGELOG-1.7.md:94]: Would you like to add this to `What's new` section, and add some descriptions?

Comment (WintonChan) [reply CHANGELOG/CHANGELOG-1.7.md:94]: ok

File: CHANGELOG/CHANGELOG-1.7.md [hunk 2]
```diff
@@ -70,6 +70,11 @@ list-watch connections on edge nodes keep available even when nodes are located
 In this release, a bunch of corner case issues are fixed and the stability is improved. And the feature maturity is now Beta.
 
 
+**Send http request to application on the edge by Router module**
+
+Router module provides a new endpoint type, servicebus. 
+It allows user call rest api of app on the edge from cloud by configuring the rule of rest to serviebus in Kubeedge.  ([#2588](https://github.com/kubeedge/kubeedge/pull/2588))
```
Comment (kevin-wangzefeng) [inline CHANGELOG/CHANGELOG-1.7.md:76]:
```suggestion
**Custom HTTP Request Routing between Cloud and Edge for Applications [Alpha]**

A new RuleEndpointType `servicebus` is added to RuleEndpoint API, to support custom http request routing between cloud and edge for applications. This simplifies the rest api access with http server on the edge while client is in the cloud.
 ([#2588](https://github.com/kubeedge/kubeedge/pull/2588))
```

**Review by fisherxu — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**